docs/Kubernetes/deployingk8s-eks.md

```diff
@@ -0,0 +1,353 @@
+--8<-- "References/abbreviations.md"
+
+# Amazon Elastic Kubernetes Service (EKS)
+
+This tutorial demonstrates how to deploy the Open AMT Cloud Toolkit on a Kubernetes cluster using EKS. Alternatively, you can also perform a simpler, test deployment using a single-node cluster locally. See [Kubernetes (K8s)](./deployingk8s.md).
```

Comment:
The thing I might question here is phrasing this as "alternatively".  We're trying to get away from phrasing things as in "you can do it this way and that way and whatever way". It's confusing to the user. 

To make this sound like a separate thing, I might say: 
To perform a simpler test deployment, use a single-mode cluster locally. See [ ].